Qadir__Work/JavaScript/Assignment21/Question2/Qadir__Assignment21__2.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="./crudOperations.js"></script>
+        <style>
+            table,
+            td,
+            th {
+                border: 1px solid;
+            }
+
+            td,
+            th {
+                text-align: center;
+                padding: 5px 100px 5px 100px;
+            }
+
+            input {
+                padding: 5px;
+                outline-color: cornflowerblue;
+            }
+
+            button, 
+            input[type="button"] {
+                border-radius: 5px;
+                padding: 7px 10px 7px 10px;
+                border: none;
+                background-color: rgb(0, 153, 255);
+                color: white;
+                font-family: 'Times New Roman', Times, serif;
+                font-weight: 700;
+                font-size: 15px;
+            }
+
+            table + button {
+                margin-top: 10px;
+                margin-left: 10px;
+            }
+        </style>
+        <title>Table Create, Update, Read and Delete</title>
+    </head>
+    <body>
+        <table id="richestManList">
+            <caption>List Of Richest Man in World</caption>
+            <tr>
```

Comment:
Wrong caption

Reply:
In Question They not given any table name or something.
Please read properly